Directory.Packages.props

```diff
@@ -16,8 +16,7 @@
       - https://github.com/dotnet/roslyn/blob/main/docs/wiki/NuGet-packages.md
       - https://learn.microsoft.com/en-us/visualstudio/productinfo/vs-servicing
     -->
-
-    <PackageVersion Include="Microsoft.CodeAnalysis.CSharp.Workspaces" Version="4.3.1" />
+    <PackageVersion Include="Microsoft.CodeAnalysis.CSharp.Workspaces" Version="4.8.0" />
```

Comment:
This means that our users must be running .NET SDK 8+. Do we want to force that before .NET 6 is out-of-support?

Reply:
This is now back to 4.3.1 and both `CodeAnalysis.CSharp` and the `Workspace` packages are next to each other under the comment so future me doesn't make the same mistake